pages/docs/concepts/asyncapi-document/AsyncAPI_structure.md

```diff
@@ -0,0 +1,384 @@
+---
```

Comment:
I think in one of the comments I saw @alequetzalli mentioning to not use underscores, so 🙏🏼 adjust the name of the file accordingly. And I'm pretty sure it should be all lowercase

Reply:
yes! what he said 😄

Reply:
**fyi, this file name should be only `structure.md`**, so that the end URL is:

`asyncapi.com/docs/concepts/asyncapi-document/structure` 

You can see why we **don't** need to repeat the word "AsyncAPI" in this file name too :)